tools/coqdep/common.ml

```diff
@@ -248,6 +248,7 @@ let parse_META package f =
     close_in ic;
     Some (f, m)
   with
+  (* with findlib >= 1.9.3 this should be Fl_metascanner.Error *)
   | Stream.Error msg -> Error.cannot_parse_meta_file package msg
```

Comment:
Our current minimum bound for findlib is 1.9.1 which doesn't have the Fl_metascanner.Error exception.
This means that when using 1.9.3 we don't catch findlib parsing errors.
Should we change our minimum findlib version?

Reply:
ok

Reply:
bumped